msal/src/Microsoft.Identity.Client/Platforms/uap/PlatformInformation.cs

```diff
@@ -112,6 +90,7 @@ public override string GetRedirectUriAsString(Uri redirectUri, RequestContext re
                 : redirectUri.OriginalString;
         }
 
+
         private static class NativeMethods
```

Comment:
> private static class NativeMethods [](start = 7, length = 35)

while not relevant to this change... move to different file... also evaluate if we need this any more. using this will potentially prevent running on anything but the x86 architecture.

Reply:
I did a quick test and this seems to work inside a WIn app. I was thinking of refactoring PlatformInformation but it would introduce quite a lot of code churn so close to the release. I think it would be safer to leave as is for now. 

I'll move NativeMethod separately.

Reply:
ok

---
In reply to: [206492047](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1153#discussion_r206492047) [](ancestors = 206492047)